.github/workflows/test.yml

```diff
@@ -12,6 +12,6 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: |
-          npm install
+          npm install --legacy-peer-deps
```

Comment:
Will look into how to update npm `package.json` to avoid peer dependancy issues, and not have to use this flag